jac-uk/admin
# #1944 Application Reminder Emails/Admin Email Signature Box

## What's included?
Closes #1944 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Go to Contacts of an exercise and check if the email signature can accept a name rather than an email address.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://user-images.githubusercontent.com/79906532/230338361-b7647a01-cb54-4e13-b3b5-f99559c4053c.mov

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
